.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+go:
+- '1.13'
+services:
+- docker
+cache:
+  directories:
+  - "$GOPATH/pkg"
+sudo: true
```

Comment:
Why do we need sudo? If we do need it, please add a comment here stating why.

Reply:
Honestly I don't know if we need it. Gonna try to remove it and see what happens.